frontends/web/package.json

```diff
@@ -9,8 +9,8 @@
     "@types/node": "^20.12.7",
     "@types/react": "18.3.3",
     "@types/react-dom": "18.3.0",
-    "@typescript-eslint/eslint-plugin": "^7.15.0",
-    "@typescript-eslint/parser": "^7.15.0",
+    "@typescript-eslint/eslint-plugin": "^7.17.0",
+    "@typescript-eslint/parser": "^7.17.0",
```

Comment:
maybe just wait until they support eslint v9?

or is there something important https://github.com/typescript-eslint/typescript-eslint/blob/main/CHANGELOG.md ?

Reply:
makes sense, I kind of picket these randomly